Cargo.toml

```diff
@@ -1,18 +1,21 @@
 [package]
 name = "clink"
+authors = ["Mary Strodl <mstrodl@csh.rit.edu>"]
 version = "0.1.1"
 edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-clap = "2.33.3"
 url = "2.2.2"
 serde_json = "1.0.71"
 serde = "1.0.130"
 ncurses = "5.101.0"
 http = "0.2.5"
+clap = { version = "3.1.6", features = ["cargo"] }
```

Comment:
we should probably decide if we want this syntax or the syntax below it...

Reply:
This syntax is what I see a lot more of, especially in the docs

Reply:
I was hoping we could use this syntax